app/Http/Controllers/V2/Dashboard/CountryDataController.php

```diff
@@ -80,21 +80,19 @@ public function getProjectData(string $uuid)
 
             if (! $project) {
                 Log::error("Project not found for project with UUID: $uuid");
+                return response()->json(['error' => 'Project not found'], 404);
             }
-            $countSitePolygons = 0;
-            if($project) {
-                $countSitePolygons = $project->getTotalSitePolygons();
-            }
+            $countSitePolygons = $project?->getTotalSitePolygons() ?? 0;
 
             $organization = $project->organisation()->first();
             if (! $organization) {
                 Log::error("Organization not found for project with ID: $project->id");
             }
 
-            $country = WorldCountryGeneralized::where('iso', $project->country)->first();
+            $country = $project ? WorldCountryGeneralized::where('iso', $project->country)->first() : null;
             $data = [
-              ['key' => 'project_name', 'title' => 'title', 'value' => $project->name ?? null],
-              ['key' => 'country', 'title' => 'Country', 'value' => $country->country ?? null],
+              ['key' => 'project_name', 'title' => 'title', 'value' => $project?->name ?? null],
+              ['key' => 'country', 'title' => 'Country', 'value' => $country?->country ?? null],
```

Comment:
Project is guaranteed not to be null here, so while the `$country?->` is still appropriate, line 92 doesn't need the ternary and line 94 should be `$project->name` instead of `$project?->name ?? null`. Also, `?? null` doesn't accomplish anything here, so this line can be simply `$country?->country`, which will evaluate to null if `$country` is null.

Reply:
fixed